Makefile

```diff
@@ -27,9 +27,11 @@ proto:
 	done
 
 test: testdeps
+	go test -i google.golang.org/grpc/...
```

Comment:
Thanks. We can if we do google-internal projects (blaze takes care of everything). We cannot if we use google machines for oss (I believe there are workarounds but I am not willing to teach every single googler that workaround when they meet this problem.). :)